Frontend/index.html

```diff
@@ -54,27 +57,27 @@
         <h1>O projekte</h1>
         <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Corrupti illo ratione aut in quaerat saepe minima atque ad, voluptatum non assumenda quisquam modi tempore numquam reprehenderit asperiores mollitia neque! Ex. Lorem ipsum dolor sit amet, consectetur adipisicing elit. Excepturi unde ad explicabo quod adipisci. Labore necessitatibus aut nihil debitis magnam maiores, sunt, sit odit est ea, beatae mollitia. Voluptatum, exercitationem.</p>
     </div>
-    <div style="line-height: 50px; font-size: 23px; background-color: rgb(145,140,140, 0.3);; color: #302d2b; text-align: center;">Na projekte pracovali:</div>
+    <div class="napr_prac">Na projekte pracovali:</div>
     <div class="about_us">
         <div class="column"><!--<img src="assets/img/12.jpg" alt="1">-->
            <h3 style="text-align: center;
            padding:  20px 0;">Maros</h3>
-            <p class="text">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Nihil maiores accusamus temporibus, quas beatae, voluptates quod sed ratione quibusdam, iusto velit a. Distinctio optio quibusdam qui nisi autem, eos voluptate.</p>
+            <p class="text">Pracoval som na koncepte tohto projektu a hardware pre Node jednotky. Mám 19 rokov a som absolvent gymnázia Jána Adama Raymana.</p>

```

Comment:
```suggestion
            <p class="text">Pracoval som na koncepte tohto projektu a hardware pre Node jednotky. Mám 19 rokov a som absolvent Gymnázia Jána Adama Raymana.</p>
```

Oficiálny názov s veľkým G.